homeassistant/components/hyundai_kia_connect/entity.py

```diff
@@ -0,0 +1,28 @@
+"""Base Entity for Hyundai / Kia Connect integration."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import BRANDS, DOMAIN, REGIONS
+
+
+class HyundaiKiaConnectEntity(CoordinatorEntity):
+    """Class for base entity for Hyundai / Kia Connect integration."""
+
+    def __init__(self, coordinator, config_entry: ConfigEntry):
+        """Initialize."""
+        super().__init__(coordinator)
+        self.config_entry = config_entry
+
+    @property
+    def unique_id(self):
```

Comment:
Typing missing. Please note, you can also use shorthand attributes as described here: <https://developers.home-assistant.io/docs/core/entity#entity-class-or-instance-attributes>